kvdb-rocksdb/src/lib.rs

```diff
@@ -650,8 +654,8 @@ impl Database {
 			.unwrap_or(0)
 	}
 
-	/// Drop a column family.
-	pub fn drop_column(&self) -> io::Result<()> {
+	/// Remove the last column family in the database. The deletion is definitive.
+	pub fn remove_last_column(&self) -> io::Result<()> {
```

Comment:
Ok I see this PR was an excuse to sneak in this change :P

Reply:
Caught red handed!